.gitlab-ci.yml

```diff
@@ -1,6 +1,6 @@
 include:
   - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-golang-unittests.yml'
+    file: '.gitlab-ci-check-commits.yml'
```

Comment:
Two comments.

wrt adding `gitlab-ci-check-commits`,
The unit tests already contain commit checks via [license_test.go](https://github.com/mendersoftware/mender-cli/blob/master/license_test.go#L22) (same way all other golang core repos). So we don't need to run this separately.

wrt removing `gitlab-ci-check-golang-unittests`,
This was added recently by me, the main motivation is to have code report integration out of the box. I suggest we restore it. Alternatively we need a publish step in this pipeline to publish the code coverage, similar to [mender repo](https://github.com/mendersoftware/mender/blob/master/.gitlab-ci.yml#L44).

Reply:
you are right wrt `gitlab-ci-check-golang-unittests`.
I was going back and forth and decided it's best to have it - but then forgot in the end! I will restore it.